src/mappings/handlers/assetHubMigrationHandler.ts

```diff
@@ -94,15 +94,9 @@ async function migrateProxiedEntities(sourceChainId: string, targetChainId: stri
 
       if (!(await Proxied.get(newId))) {
         await Proxied.create({
+          ...ent,
```

Comment:
for me direct property redefinition looks better than spread, you're hiding a lot of necessary information